apollo-router/src/query_planner/query_planner_service.rs

```diff
@@ -324,6 +327,11 @@ impl QueryPlannerService {
                 "Number of query plans evaluated for a query before choosing the best one",
                 evaluated_plan_count
             );
+            u64_histogram!(
+                "apollo.router.query_planning.plan.evaluated_query_options",
```

Comment:
to match the one right above this
```suggestion
                "apollo.router.query_planning.plan.evaluated_options",
```